packages/patternfly-4/react-charts/src/components/ChartStack/examples/ChartStack.md

```diff
@@ -90,13 +89,12 @@ import { Chart, ChartAxis, ChartStack, ChartThemeColor } from '@patternfly/react
 </div>
 ```
 
-## Multi-color (ordered), horizontal stacked bar chart with bottom-left aligned legend
-This demonstrates an alternate way of applying tooltips using data labels
-```js
+```js title=Multi-color-(ordered),-horizontal-stacked-bar-chart-with-bottom-left-aligned-legend
```

Comment:
Why do some titles have dashes instead of white space, while others do not?

Reply:
There are a couple examples that start off as `Bottom-left` and `Multi-color` -- note the intentional dash

Reply:
White space is not supported in code block attributes by `gatsby-mdx`. We use dashes and then string replace them with spaces in our `gatsby-theme-patternfly-org`.